test/e2e/app-dir/app-css/index.test.ts

```diff
@@ -50,16 +50,17 @@ createNextDescribe(
           )
         })
 
-        it('should support external css imports', async () => {
+        it.only('should support external css imports', async () => {
```

Comment:
Was this `.only` here because you were trying to run this test in isolation? Should this be removed?